docs/Server/API/access_model.md

```diff
@@ -17,12 +17,35 @@ Any client, regardless of authentication, is able to READ a dataset with the
 `access` property set to `"public"`. Only the owner of the dataset, or a user with
 the administrator *role*, can UPDATE or DELETE the dataset.
 
+## User Identity
+
+The Pbench Server works with an external OIDC identity provider, for example
+Red Hat, Google, or GitHub SSO. The Pbench Server authenticates user access using
```

Comment:
Instead of "works with", perhaps something like "is configured to use an" (or just "uses").  (And, I think I would be inclined to insert a "the" between "example" and "Red Hat".)